log-analyzer/features/org.wso2.carbon.la.apim.feature/src/main/resources/gadgets/LogAPIMArtifactDeleted/js/moment.js

```diff
@@ -0,0 +1,2808 @@
+//! moment.js
```

Comment:
Hi Sinthuja,

Yes, there is minified version for moment.js and we have replaced them in following commits, 

https://github.com/wso2/shared-analytics/pull/24/files#diff-2df137c6c122e1acbc40d4db4a831c01L1

https://github.com/wso2/shared-analytics/pull/5/files

Thanks,
SajithD